prefetch.bs

```diff
@@ -290,12 +296,12 @@ The user agent may [=prefetch record/cancel and discard=] records from the [=Doc
     1. Let |recordToUse| be |exactRecord| if |exactRecord| is not null, otherwise |inexactRecord|.
     1. If |recordToUse| is not null:
         1. Let |currentTime| be the [=current high resolution time=] for |navigable|'s [=navigable/active document=].
-        1. If |recordToUse|'s [=prefetch record/expiry time=] is less than |currentTime|, return null.
+        1. If |recordToUse|'s [=prefetch record/expiry time=] is less than |currentTime|, return null and [=trigger a prefetch status updated event=] given |navigable|, |recordToUse|, and "{^speculation.PreloadingStatus/failure^}" status
```

Comment:
It's generally confusing to do "return null and do X", because after returning no more steps can happen. I'd suggest nested lists instead, i.e.

```
1. If ...:
    1. Trigger a prefetch status updated event given ...
    2. Return null.

(Here and below.)